qiita_plugins/biom_type/biom_type_plugin/create.py

```diff
@@ -0,0 +1,110 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from os.path import join, basename
+
+from json import loads
+
+from biom import load_table
+from biom.util import biom_open
+from biom.exception import TableException
+from qiita_client import format_payload
+
+
+def create_artifact(qclient, job_id, parameters, out_dir):
+    """Validate and fix a new BIOM artifact
```

Comment:
Yeah, I guess you are right I was trying to show that this are the steps performed before the artifact is actually created in the system (that happens once the job is completed) but I think you're right and makes sense to change it to `validate`.